src/main/java/module/specialEvents/SpecialEventsTable.java

```diff
@@ -82,26 +42,17 @@ public boolean isCellEditable(int rowIndex, int mColIndex) {
 	}
 
 	private String getTacticToolTipText(MatchRow row, boolean homeTeam) {
-		int tactic = (homeTeam) ? row.getMatch().getHostingTeamTactic() : row.getMatch()
-				.getVisitingTeamTactic();
-		switch (tactic) {
-		case IMatchDetails.TAKTIK_NORMAL:
-			return getLangStr("ls.team.tactic.normal");
-		case IMatchDetails.TAKTIK_PRESSING:
-			return getLangStr("ls.team.tactic.pressing");
-		case IMatchDetails.TAKTIK_KONTER:
-			return getLangStr("ls.team.tactic.counter-attacks");
-		case IMatchDetails.TAKTIK_MIDDLE:
-			return getLangStr("ls.team.tactic.attackinthemiddle");
-		case IMatchDetails.TAKTIK_WINGS:
-			return getLangStr("ls.team.tactic.attackonwings");
-		case IMatchDetails.TAKTIK_CREATIVE:
-			return getLangStr("ls.team.tactic.playcreatively");
-		case IMatchDetails.TAKTIK_LONGSHOTS:
-			return getLangStr("ls.team.tactic.longshots");
-		default:
-			return "unknown";
-		}
+		int tactic = (homeTeam) ? row.getMatch().getHostingTeamTactic() : row.getMatch().getVisitingTeamTactic();
+		return switch (tactic) {
+			case IMatchDetails.TAKTIK_NORMAL -> getLangStr("ls.team.tactic.normal");
```

Comment:
In case this helps, the “pattern” I have been using for this kind of situation is to use the enum value as part of the label key, cf. https://github.com/akasolace/HO/blob/master/src/main/java/module/series/promotion/PromotionInfoPanel.java#L138.  This avoids the conditional approach.

Reply:
I am not sure of what you want me to do here? A different way to get the tactic or to change the switch?